### ISSUE TITLE
Use setup.cfg to use README.md for PyPI description

### ISSUE THREAD

**Issue by dhimmel** [dhimmel/hetio#7]
Currently, we have to use `README.rst` for the `long_description` in setuptools to created the description for PyPI.

Accoridng to [this tutorial](https://www.codementor.io/arpitbhayani/host-your-python-package-using-github-on-pypi-du107t7ku), we may be able to create a `setup.cfg` file containing:

```cfg
[metadata]
description-file = README.md
```

That would be awesome. This rst format and duplicating the README has been a real annoyance. Will wait on the merging of https://github.com/dhimmel/hetio/pull/6, so this doesn't cause any conflicts.

**Comment by dhimmel** [comment]
Just tried the method from [the tutorial](https://www.codementor.io/arpitbhayani/host-your-python-package-using-github-on-pypi-du107t7ku) and no long description showed up on [test PyPI](https://testpypi.python.org/pypi/hetio).

Therefore, It seems like there is no easy way to use a markdown README for PyPI. Solutions involve requiring pandoc locally, which is a heavy dependency.

I think the best thing to do is just make the rst README for both GitHub and PyPI long_description. What an annoyance!

**Comment by dhimmel** [comment]
See https://github.com/dhimmel/hetio/commit/0faba3459cb45423375063a1ceab1d916c3ce01b. I ended up using pandoc via a `subprocess` call.

I think this is a decent workaround. If pandoc isn't installed, then `long_description` is None. This would only matter for the installation that is getting uploaded to PyPI. I added a pandoc dependency on Travis, so if we switch to [Travis deployment](https://docs.travis-ci.com/user/deployment/pypi/), then we should be all good.

**Comment by dhimmel** [comment]
See https://github.com/dhimmel/hetio/commit/8f74660866b322d3a68a556aad5eec7785c9ee39. I ended up using pandoc via a `subprocess` call.

I think this is a decent workaround. If pandoc isn't installed, then `long_description` is None. This would only matter for the installation that is getting uploaded to PyPI. I added a pandoc dependency on Travis, so if we switch to [Travis deployment](https://docs.travis-ci.com/user/deployment/pypi/), then we should be all good.
